pkg/kubectl/cmd/set/set_env.go

```diff
@@ -215,7 +215,13 @@ func (o *EnvOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []stri
 	o.Resources = resources
 	o.Cmd = cmd
 
-	o.PrintFlags.Complete(o.DryRun)
+	if o.DryRun {
+		// TODO(juanvallejo): This can be cleaned up even further by creating
```

Comment:
Hmmm... this is worth a bigger discussion. @deads2k I'm slightly skeptical adding a whole Printer just to pass the `(dry run)` message, seems a slight overkill. Why not just embed `DryRun` inside the printer, but leave the wiring to a user? Let's discuss this on Monday.